src/Bridge/Symfony/HttpFoundation/ResponseHeadersAndCookiesProcessor.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace K911\Swoole\Bridge\Symfony\HttpFoundation;
+
+use Swoole\Http\Response as SwooleResponse;
+use Symfony\Component\HttpFoundation\Response as HttpFoundationResponse;
+
+final class ResponseHeadersAndCookiesProcessor implements ResponseProcessorInterface
```

Comment:
I'd name it like `ResponseHeadersAndStatusProcessor` since cookies are represented by headers

Reply:
done